app/static/src/components/Shopcart.vue

```diff
@@ -44,11 +44,11 @@
     <products
       v-bind="$attrs"
       v-model="addedProducts">
-      <button class="btn btn-primary" @click="updateCart" id="update-btn">Update</button>
     </products>
+    <button class="btn btn-primary" @click="updateCart" id="update-btn">Update</button>
```

Comment:
This button should be within the products element.

Reply:
done!